lib/Wireguard/WGmeta/Wrapper/Config.pm

```diff
@@ -874,14 +933,14 @@ Two strings. With example values given in the parameters this would be 'This' an
 
 =cut
 sub split_and_trim($line, $separator) {
-    return map {s/^\s+|\s+$//g;
+    return map {s/^\s+|\s+$//;
```

Comment:
hmmm will this remove extra space at the start AND at the end still?

Reply:
no :) - reverted